### PR TITLE
[Test/Style] Add tox tests and style to PEP8 format

### PR DESCRIPTION
**Pull request by muneeb-jan**

### CODE REVIEW

File: src/globalmon/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-# This file is important for setup.py to find modules.
\ No newline at end of file
+# This file is important for setup.py to find modules.
```
Comment (anton-sidelnikov) [inline src/globalmon/__init__.py:1]: no need to use this comment, anyone from python world should know the reason of this file

File: tests/test_log_to_statsd.py [hunk 2]
```diff
@@ -15,8 +15,10 @@ def test_log_to_statsd(self, mock_stats_client):
             }
         }
         log_to_statsd(mock_client, 'prefix', results)
-        mock_client.timing.assert_called_once_with('stats.timer.prefix.example_service.http://example.com.200', 100)
-        mock_client.incr.assert_called_once_with('stats.counter.prefix.example_service.http://example.com.200')
+        mock_client.timing.assert_called_once_with(
+            'stats.timer.prefix.example_service.http://example.com.200', 100)
+        mock_client.incr.assert_called_once_with(
+            'stats.counter.prefix.example_service.http://example.com.200')
```
Comment (anton-sidelnikov) [inline tests/test_log_to_statsd.py:21]: there is no error with this double slashes in statsd? did you check it?

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+[flake8]
+# E123, E125 skipped as they are invalid PEP-8.
+show-source = True
+ignore = E123,E125,W503,H101
+builtins = _
+exclude=.venv,.git,.tox,dist,doc,*lib/python*,*egg,build,git_repos
+
+[stestr]
+test_path = ./tests
```
Comment (anton-sidelnikov) [inline tox.ini]: add endline please

**Review by anton-sidelnikov — DISMISSED** [review]

File: tests/test_log_to_statsd.py (hunk 2 above)
Comment (anton-sidelnikov) [reply tests/test_log_to_statsd.py:21]: maybe better to remove part with http:// ?

Comment (anton-sidelnikov) [reply tests/test_log_to_statsd.py:21]: also statsd will split example and com

File: tox.ini (hunk 3 above)
Comment (muneeb-jan) [reply tox.ini]: Done

File: tests/test_log_to_statsd.py (hunk 2 above)
Comment (muneeb-jan) [reply tests/test_log_to_statsd.py:21]: Not when i tested.

File: src/globalmon/__init__.py (hunk 1 above)
Comment (muneeb-jan) [reply src/globalmon/__init__.py:1]: I prefer code/type hints.. improves readability.

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Comment by Persimmonboy** [comment]
gate

**Comment by Persimmonboy** [comment]
regate

**Pull request merged**